index.d.ts

```diff
@@ -680,8 +680,19 @@ export interface Interpolator {
 
 export class ResourceStore {
   constructor(data: Resource, options: InitOptions);
+
   public data: Resource;
   public options: InitOptions;
+
+  /**
+   * Gets fired when resources got added
+   */
+  on(event: 'added', callback: (lng: string, ns: string) => void): void;
```

Comment:
Sorry, I just noticed, why not combine these as:

```diff
- on(event: 'added', callback: (lng: string, ns: string) => void): void;
- on(event: 'removed', callback: (lng: string, ns: string) => void): void;
+ on(event: 'removed' | 'added', callback: (lng: string, ns: string) => void): void;
```

Reply:
Thanks, fixed